torchrl/objectives/ppo.py

```diff
@@ -8,7 +8,7 @@
 
 from copy import deepcopy
 from dataclasses import dataclass
-from typing import Tuple
+from typing import Sequence, Tuple
```

Comment:
Perhaps let's restrict it to tuple? Having a sequence seems just a tiny bit more complex, and tuples seem to be the right class for this.

```suggestion
from typing import Tuple
```